tmva/sofie/test/Conv1dModelGenerator.py

```diff
@@ -132,14 +132,20 @@ def main():
       torch.save({'model_state_dict':model.state_dict()}, name + ".pt")
 
    if saveOnnx:
-      torch.onnx.export(
-         model,
-         xinput,
-         name + ".onnx",
-         export_params=True,
-         dynamo=True,
-         external_data=False
-      )
+      #check torch version
+      v = torch.__version__
+      from packaging.version import Version
```

Comment:
`packaging` is not a standard Python module, https://pypi.org/project/packaging/ . If we need to use it in these tests, it must be added to `requirements.txt` before merging this PR. Even better, you could think about not using it altogether